### ISSUE TITLE
[Bug]: Showing a second browser window after destroying the first causes the renderer to hang.

### ISSUE THREAD

**Issue by Avlyssna** [electron/electron#29050]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.0.6

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Home 20H2

### What arch are you using?

x64

### Expected Behavior

1. I create two `BrowserWindow` instances in the application `ready` event. Both specify `show: false`.
2. I `.show` one of the instances as a splash screen within the `ready-to-show` event and wait 5 seconds.
3. I `.destroy` the splash screen's window and `.show` the application window.
4. The application window should be focused and allow use.

### Actual Behavior

1. I create two `BrowserWindow` instances in the application `ready` event. Both specify `show: false`.
2. I `.show` one of the instances as a splash screen within the `ready-to-show` event and wait 5 seconds.
3. I `.destroy` the splash screen's window and `.show` the application window.
4. The application window's renderer seems to be frozen and displays only the original HTML rendering.

When this happens, I can try to type, but it won't render. However, when I blur the window by clicking on the taskbar and re-focus the application, my typed text shows. It appears that the window is functioning - just the renderer is not showing any changes.

Funny enough, this bug won't happen if I destroy the splash window **after** showing the application window. Maybe it's having trouble changing the focus from a destroyed window (the splash) when I call `.show` for the application window?

I can reproduce this every launch, so if there's problem reproducing on your end, let me know.

### Testcase Gist URL

https://gist.github.com/Avlyssna/1fe5b8f7cac8242401d2c3d7a4f20d0e

**Comment by Prinzhorn** [comment]
I think this might be related to #28255, #28603 and maybe #27353 
There seems to be some general issues with rendering after calling `show` on Windows.

**Comment by Avlyssna** [comment]
I'd say #27353 is probably the closest to my specific use-case. While it could be something to do with the `show` function in general, it seems odd to me that I'm able to fully bypass the issue if I call `destroy` on the first window _**after**_ I show the main one. Maybe something with that order of calls is inadvertently fixing it.

**Comment by jackple** [comment]
I have a similar problem on Win10.
User click some where, logger works fine, but Dom is frozen.

**Comment by nornagon** [comment]
Hm, I can't repro this. The gist you posted was incomplete, but here's a gist based on it that does run with [Electron Fiddle](/electron/fiddle): https://gist.github.com/9bc40bf4957f5d459292f055285177d5

The `<input>` seems to work fine.

**Comment by Avlyssna** [comment]
> Hm, I can't repro this. The gist you posted was incomplete, but here's a gist based on it that does run with [Electron Fiddle](/electron/fiddle): https://gist.github.com/9bc40bf4957f5d459292f055285177d5
> 
> The `<input>` seems to work fine.

Perhaps it's because you used Electron Fiddle, but using the `electron .` script from the folder still reproduces it on a completely different system. I've updated the gist to base off yours with a slightly different `package.json`. I've also attached a clip of the bug in action. It doesn't update the renderer contents until the windows is dragged (or minimized).

https://user-images.githubusercontent.com/7981517/125381038-d67bd900-e358-11eb-9c0f-b5b560c9a916.mp4
